### ISSUE TITLE
xacro warnings

### ISSUE THREAD

**Issue by Tiryoh** [rt-net/raspimouse_sim#18]
* ENV
  * Ubuntu 16.04
  * ROS Kinetic
  * kinetic-devel

```
ubuntu@ubuntu:~/catkin_ws/src/raspimouse_sim$ rosrun xacro xacro raspimouse_gazebo/materials/sample_maze.world.xacro 
xacro: Traditional processing is deprecated. Switch to --inorder processing!
To check for compatibility of your document, use option --check-order.
For more infos, see http://wiki.ros.org/xacro#Processing_Order
deprecated: xacro tags should be prepended with 'xacro' xml namespace.
Use the following script to fix incorrect usage:
        find . -iname "*.xacro" | xargs sed -i 's#<\([/]\?\)\(if\|unless\|include\|arg\|property\|macro\|insert_block\)#<\1xacro:\2#g'
when processing file: raspimouse_gazebo/materials/sample_maze.world.xacro

...

xacro.py is deprecated; please use xacro instead
```

**Comment by Tiryoh** [comment]
.world file is an SDF formatted file, so this warning seems to be ignored?
SDF file and URDF file are similar, but not the same.

**Comment by Tiryoh** [comment]
solved on 363dee3f3bcd15ec4763055daf0e210c05226b9e
